development/cleanup.sh

```diff
@@ -22,16 +21,17 @@ end;
 drop user ${UT3_OWNER} cascade;
 drop user ${UT3_RELEASE_VERSION_SCHEMA} cascade;
 drop user ${UT3_TESTER} cascade;
+drop user ${UT3_TESTER_HELPER} cascade;
 drop user ${UT3_USER} cascade;
 
 begin
   for i in (
-    select decode(owner,'PUBLIC','drop public synonym "','drop synonym "'||owner||'"."')|| synonym_name ||'"' drop_orphaned_synonym, owner||'.'||synonym_name syn from dba_synonyms a
```

Comment:
Can we keep that functionality?

Reply:
Hmm good catch I dont remember changing this. I will double check whats going on.

Reply:
I have branched out of random order , must some changes there that were not pulled back.